core/src/com/unciv/logic/automation/WorkerAutomation.kt

```diff
@@ -14,8 +14,7 @@ class WorkerAutomation(val unit: MapUnit) {
     fun automateWorkerAction() {
         val enemyUnitsInWalkingDistance = unit.movement.getDistanceToTiles().keys
                 .filter {
-                    it.militaryUnit != null && it.militaryUnit!!.civInfo != unit.civInfo
-                            && unit.civInfo.isAtWarWith(it.militaryUnit!!.civInfo)
+                    it.militaryUnit?.civInfo?.isAtWarWith(unit.civInfo) ?: false
```

Comment:
Another love which I think is much more readable as 'if's

Reply:
will do